### DISCUSSION TITLE
Direct thumbnail selection

### DISCUSSION THREAD

**Discussion by fade0ff** [mifi/lossless-cut#1099] [Ideas]
I'm aware of the possibility to add an external image as thumbnail, but given that the tool already has access to all necessary resources, it would be much more convenient if there was an option to use the currently displayed frame as embedded thumbnail for the exported video. Any chance this could happen?

**Comment by mifi** [comment]
Thanks for the suggestion. I agree this is a cool feature to have. Created a feature request from it: #1100
